app/javascript/dashboard/components/widgets/WootWriter/AudioRecorder.vue

```diff
@@ -134,6 +132,11 @@ export default {
   },
   methods: {
     deviceReady() {
+      if (this.player.record().engine instanceof OpusRecorderEngine) {
+        if (this.audioRecordFormat === AUDIO_FORMATS.WEBM) {
```

Comment:
```suggestion
        if (this.audioRecordFormat === AUDIO_FORMATS.WAV) {
```
Let's add a new value for the `wav` format - https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/shared/constants/messages.js#L84

```
export const AUDIO_FORMATS = {
  WEBM: 'audio/webm',
  OGG: 'audio/ogg',
  WAV: 'audio/wav',
};
```